roles/grubcmdline/meta/main.yml

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
Do we need it for a role in a collection?

Reply:
Don't think so, I will remove.

Reply:
done